lib/js/components/Skeleton/Skeleton.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+	<prime-skeleton
+		width="100%"
+		height="100%"
```

Comment:
why we don't expose `width` and `height` as props?

Reply:
We try to avoid hardcoded sizes, so having skeleton fill the available place sounds like the cleanest approach. Parent can use flexbox etc. to have it sized appropriately. We can add props when there is an actual need.

Reply:
![Zrzut ekranu 2024-07-26 o 08 14 15](https://github.com/user-attachments/assets/b839974f-be58-4e0a-9149-bc41957fd617)

IMO, there is an acttual need for this prop in OverlayHeader. Instead of creating a <div> wrapper for every line I could set `width="30%"` on a first line.